### PR TITLE
Adapt template with changes

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Fix: #2 

---

- [x] TS support,~~compiled with babel preferably~~
- [x] Remove unnecessary modules
- [x] ESLint config (**TBD**) - Remove the current TSLint config (deprecated)
- [x] Jest config (**TBD**)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,14 @@
 {
-  "name": "ts-express-jest",
-  "version": "1.0.0",
```
Comment (sauravhiremath) [inline package.json:3]: Let's keep the version 0.0.0 till the initial release

File: src/app.ts [hunk 2]
```diff
@@ -2,22 +2,31 @@ import * as bodyParser from "body-parser";
 import express from "express";
 import { Routes } from "./routes";
 
+/**
+ * @class
```
Comment (sauravhiremath) [inline src/app.ts:6]: Any specific reason the comments are removed? It doesn't really make a difference, you can carry on however it suits you for now

File: package.json (hunk 1 above)
Comment (YashKumarVerma) [reply package.json:3]: Sure, I think the PR changes 1.0.0 to 0.0.0 so no changes needed here right?

File: src/app.ts (hunk 2 above)
Comment (YashKumarVerma) [reply src/app.ts:6]: added them back

**Pull request merged**